srcpkgs/sov/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'sov'
+pkgname=sov
+version=0.63
+revision=1
+build_style=meson
+hostmakedepends="pkg-config cmake wayland-devel"
+makedepends="wayland-protocols wayland-devel freetype-devel"
+depends="wayland"
```

Comment:
this should be autodetected, remove this line

Reply:
The `depends` assignment?

Reply:
yes